src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealFormMocks.tsx

```diff
@@ -0,0 +1,74 @@
+import {
```

Comment:
Do you want to change this extension to `.ts` since it doesn't export components or use JSX?